README.md

```diff
@@ -101,11 +100,11 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
+[X] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.getElementsByClassName("post")[0].innerHTML`
-[] `document.getElementsByClassName("post").innerHTML`
+[X] `$(".post").html()`
+[X] `document.getElementsByClassName("post")[0].innerHTML`
+[X] `document.getElementsByClassName("post").innerHTML`
 ```
```

Comment:
That last one won't work because you're calling `.innerHTML` on an array of DOM elements. It works with the brackets in the fourth answer because you are running `.innerHTML` on a particular element in that array.